### DISCUSSION TITLE
Perlin Noise and pattern generator?

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#540] [Ideas]
It would be cool if we could generate noise and pattern (like brick pattern) and if it could be animatable like the ability to move noise or change the seed. That would be cool

**Comment by Lioncub901** [comment]
oof I thought property noise was Perlin noise, lol

**Comment by nicoleham** [comment]
yeah it might need a rename.. the current 'noise' effect is closer to a boil or wiggle 😭

**Comment by kavlor** [comment]
I think a noise effect would be pretty easy to implement plus quite useful. Maybe Emboss too as that can create nice texturing with noise.

**Reply by Lioncub901** [reply]
I know that the apple API has a noise function that can be used.
